src/browser/components/RequestsProcessor/GetStorage/index.js

```diff
@@ -17,27 +19,31 @@ export default function makeStorageComponent(storageActions) {
 
     // Rename arguments given by the user
     withProps(({ args }) => {
-      const options = args[2] || {};
-      return {
-        scriptHash: args[0],
-        storageKey: args[1],
-        encode: !!options.encode
-      };
+      const result = pick(args[0], CONFIG_KEYS);
+      result.index = result.key; // key is reserved in React props, so we map it to index
```

Comment:
Extremely minor nitpick, but if we call `Object.keys(result)` before and after this line, it's possible that "key" won't be in there before, but "index" will be in there after.  The value will still be `undefined` though, so it's probably not worth worrying about here unless we're validating user input in some sense that depends on that behavioral difference.

Reply:
More like:

```js
if (result.key) {
  result.index = result.key;
  delete result.key;
}
```

But not really important I don't think.